pkg/tagger/collectors/docker_main.go

```diff
@@ -44,10 +44,18 @@ func (c *DockerCollector) Detect(out chan<- []*TagInfo) (CollectionMode, error)
 	c.stop = make(chan bool)
 	c.infoOut = out
 
-	// viper lower-cases map keys, so extractor must lowercase before matching
-	c.labelsAsTags = config.Datadog.GetStringMapString("docker_labels_as_tags")
-	c.envAsTags = config.Datadog.GetStringMapString("docker_env_as_tags")
+	// We lower-case the values collected by viper as well as the ones from inspecting the labels of containers.
+	labelsList := config.Datadog.GetStringMapString("docker_labels_as_tags")
+	for label, value := range labelsList {
+		labelsList[strings.ToLower(label)] = value
```

Comment:
@CharlyF the code is working but I guess you would also want to get rid of the previous capitalized value?
Here you'll still store both, is it wanted?

you'll want to add a `delete statement`